File: 0075_Sort Colors/solution.go
func sortColors(nums []int)  {
    tmp := 0
    n := len(nums)
    for i := 0; i < n; i++ {
        for j := i + 1; j < n; j++ {
            if nums[i] > nums[j] {
                tmp = nums[i]
                nums[i] = nums[j]
                nums[j] = tmp
            }
        }
    }
}

File: 0075_Sort Colors/solution.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        tmp = 0
        n = len(nums)
        for i in range(n):
            for j in range(i + 1, n):
                if nums[i] > nums[j]:
                    tmp = nums[i]
                    nums[i] = nums[j]
                    nums[j] = tmp